File: src/main/java/de/cplaiz/activecraftcore/messages/Reasons.java
package de.cplaiz.activecraftcore.messages;

public class Reasons extends MessageCollection {

    public static String HACKING() {
        return getRawMessage("reason.hacking");
    }

    public static String BOTTING() {
        return getRawMessage("reason.botting");
    }

    public static String UNAUTHORIZED_ALTERNATE_ACCOUNT() {
        return getRawMessage("reason.unauthorized-alt-account");
    }

    public static String SPAM() {
        return getRawMessage("reason.spam");
    }

    public static String ABUSIVE_LANGUAGE() {
        return getRawMessage("reason.abusive-language");
    }

    public static String STEALING() {
        return getRawMessage("reason.stealing");
    }

    public static String GRIEFING() {
        return getRawMessage("reason.griefing");
    }

    public static String MODERATOR_BANNED() {
        return getRawMessage("reason.moderator-banned");
    }

    public static String MODERATOR_WARNED() {
        return getRawMessage("reason.moderator-warned");
    }

    public static String MODERATOR_KICKED() {
        return getRawMessage("reason.moderator-kicked");
    }
}

File: src/main/java/org/activecraft/activecraftcore/messages/messagecollections/Reasons.kt
package org.activecraft.activecraftcore.messages.messagecollections

import org.activecraft.activecraftcore.messages.MessageSupplier

class Reasons(val messageSupplier: MessageSupplier) {
    val hacking: String get() = messageSupplier.getRawMessage("reason.hacking")
    val botting: String get() = messageSupplier.getRawMessage("reason.botting")
    val unauthorizedAlternateAccount: String get() = messageSupplier.getRawMessage("reason.unauthorized-alt-account")
    val spam: String get() = messageSupplier.getRawMessage("reason.spam")
    val abusiveLanguage: String get() = messageSupplier.getRawMessage("reason.abusive-language")
    val stealing: String get() = messageSupplier.getRawMessage("reason.stealing")
    val griefing: String get() = messageSupplier.getRawMessage("reason.griefing")
    val moderatorBanned: String get() = messageSupplier.getRawMessage("reason.moderator-banned")
    val moderatorWarned: String get() = messageSupplier.getRawMessage("reason.moderator-warned")
    val moderatorKicked: String get() = messageSupplier.getRawMessage("reason.moderator-kicked")
}